### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by popravich**
## What do these changes do?

* Fix some rst syntax (ClientSession class was malformed);

* Fix some styling;

* Amended few text lines;

### CODE REVIEW

File: docs/glossary.rst [hunk 1]
```diff
@@ -8,6 +8,13 @@
 .. if you add new entries, keep the alphabetical sorting!
```
Comment (popravich) [inline docs/glossary.rst:8]: Do we need this comment, 'cause we can use `:sorted:` option?

File: docs/faq.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ Try something like::
 
 Where first argument, star, means catch any possible method
 (*GET, POST, OPTIONS*, etc), second matching ``url`` with desired prefix,
-third - handler.
+third -- handler.
```
Comment (asvetlov) [inline docs/faq.rst:26]: And why have you added `--` here?

File: docs/client.rst [hunk 3]
```diff
@@ -79,7 +79,7 @@ that case you can specify multiple values for each key::
                            params=params) as r:
         assert r.url == 'http://httpbin.org/get?key=value2&key=value1'
 
-You can also pass :class:`str` content as param, but beware - content
+You can also pass :class:`str` content as param, but beware -- content
```
Comment (asvetlov) [inline docs/client.rst:82]: I just curious: what's wrong with `--`?

File: docs/glossary.rst (hunk 1 above)
Comment (asvetlov) [inline docs/glossary.rst:8]: I have no idea what `:sorted:` does.

Comment (popravich) [inline docs/glossary.rst:8]: it sorts entries in alphabetical order)

File: docs/client.rst (hunk 3 above)
Comment (popravich) [inline docs/client.rst:82]: "-" basically looks to short and double dashes converted into a real wide dash "–"

**Comment by asvetlov** [comment]
LGTM, please merge.

**Pull request merged**